### ISSUE TITLE
Support tiddlywiki5 syntax to markdown

### ISSUE THREAD

**Issue by somenxavier** [jgm/pandoc#5356]
Can you please implement conversion from [tiddlywiki](http://tiddlywiki.com) format to markdown

**Comment by mb21** [comment]
Apparently, they call their syntax [WikiText](https://tiddlywiki.com/static/WikiText.html)

**Comment by Atlas48** [comment]
Seconded, as per my issue, #6602.

**Comment by dbobak** [comment]
I second this also. That's quite unique piece of software; very convenient in everyday use. And it's interoperability is great - in most cases, it's just one html file.

@mb21, they call they syntax WikiText, but it has some differences from, for example, MediaWiki.

**Comment by RamXX** [comment]
It would be great to support this. I've moved to [Drift](https://akhater.github.io/drift/) (a TiddlyWiki custom distribution) exclusively for my personal and professional information management, but for very large amounts of text, I still prefer to use [IA Writer](https://ia.net/writer) (Markdown). Cut and paste is not ideal but I still think it's acceptable if we could get this conversion natively. Thank you!

**Comment by somenxavier** [comment]
Can I offer 10 bucks to that.

**Comment by ickc** [comment]
I certainly will offer 10 bucks for all spammers to shut up.

**Comment by tarleb** [comment]
I agree that I'd rather not read spammy messages. I'd also like to ask everybody to keep it civil, please.

**Comment by somenxavier** [comment]
> I agree that I'd rather not read spammy messages. I'd also like to ask everybody to keep it civil, please.

I'm not spammer. I'm just a tiddlywiki user.
I just wanted to donate some bucks to speed the solution of this bug but the responses I saw.... it disappointed me.

**Comment by tarleb** [comment]
I believe that your comment was done in good faith, but it doesn't help here. Some suggestions:

1. Want to help financially? Make an unconditional donation.
2. Want to pay for the feature? Ask for a quote.
3. Show appreciation on a budget? Offer to buy a drink or just say "thanks"!

Frankly, expecting preferable treatment for a tip is borderline insulting.

**Comment by joshkunz** [comment]
As a heads up: I've started on working on a TiddlyWiki **writer**. Looking at the history, it seems like new writers are added as a single commit, so I'll wait to open a PR till it's complete.

**Comment by jeffrey4l** [comment]
For who wanna convert markdown to tiddlywiki, i implement a lua plugin to do this. Check following 

https://github.com/jeffrey4l/pandoc-addons

**Comment by toadico** [comment]
From a comment it does sound like this is in progress, thank you for that. 

Until that is complete, have found a work around. Import the markdown plugin to your tiddlywiki, and then convert to markdown format. Just make sure to set type to Markdown.

**Comment by marph91** [comment]
> Can you please implement conversion from [tiddlywiki](http://tiddlywiki.com) format to markdown

As another workaround for this particular conversion, you could try [jimmy](https://marph91.github.io/jimmy/formats/tiddlywiki/). It implements a [custom conversion based on pyparsing](https://github.com/marph91/jimmy/blob/2f313e148cf4826dc7345eab8b93253e4207a805/src/markdown_lib/tiddlywiki.py).

If a wikitext reader with the same capabilities would be available in Pandoc, I would prefer to use it (since other conversions are based on Pandoc already). Unfortunately I didn't have the time/motivation to learn Haskell or Lua, yet.
